README.md

```diff
@@ -496,6 +560,7 @@ Supported tasks:
     examples
     format
     init
+    saucelab-tests
```

Comment:
Should probably be "saucelab**s**-tests", right? Same goes for libraries/config/etc.